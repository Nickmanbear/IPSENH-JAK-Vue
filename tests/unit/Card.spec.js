// import { shallowMount } from '@vue/test-utils';
// import Card from '@/components/Card.vue';
//
// describe('Card.spec.js', () => {
//   let cmp;
//
//   beforeEach(() => {
//     cmp = shallowMount(Card, {
//       propsData: {
//         card: {
//           id: 5,
//           columnId: 1,
//           name: 'test card',
//           description: 'test description for the card',
//         },
//       },
//     });
//   });
//
//   it('equals name to "test card"', () => {
//     expect(cmp.find('p').text()).toEqual('test card');
//   });
//
//   it('has the expected html structure', () => {
//     expect(cmp.element).toMatchSnapshot();
//   });
// });
