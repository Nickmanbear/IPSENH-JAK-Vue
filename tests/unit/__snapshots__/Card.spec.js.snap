// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Card has the expected html structure 1`] = `
<div
  id="card"
>
  <button
    id="close"
  >
    Ã—
  </button>
   
  <div>
    <h3>
      test card
    </h3>
     
    <!---->
    
    |
    
    <span
      id="assign-user-button"
    >
      <span>
        +
      </span>
       
      <span>
         Assign user
      </span>
    </span>
     
    <!---->
  </div>
   
  <div>
    <span>
      Priority: 
    </span>
     
    <span>
      high
    </span>
     
    <!---->
     
    <span>
       - 
    </span>
     
    <span>
      Points: 
    </span>
     
    <span>
      5
    </span>
     
    <!---->
     
    <p>
      <strong>
        Description
      </strong>
    </p>
     
    <div>
      <!---->
       
      <p>
        test description for the card
      </p>
    </div>
     
    <!---->
  </div>
</div>
`;
